### ISSUE TITLE
pubnub and applovin marked as porn

### ISSUE THREAD

**Issue by josemarq** [nextdns/metadata#426]
Hello, I only want to report pubnub.com and applovin.com is marked as porn category incorrectly.

Please check. Thanks

**Comment by crssi** [comment]
None of those are marked as porn. Check again.

**Comment by josemarq** [comment]
I put this into my whitelist because for me was blocked a few days a go. 
![Screenshot_20201007-150450](https://user-images.githubusercontent.com/315641/95334582-9ad64f00-08ae-11eb-990f-567db7dd0455.png)

**Comment by josemarq** [comment]
I has been added the site to a whitelist because was in the category a few days ago.

Please check this:

![Screenshot_20201007-150948](https://user-images.githubusercontent.com/315641/95335109-4384ae80-08af-11eb-907d-7c40f76556b6.png)
![Screenshot_20201007-150927](https://user-images.githubusercontent.com/315641/95335114-441d4500-08af-11eb-8988-48c3eb05f7b7.png)

**Comment by crssi** [comment]
Did you check today before posting issue?

**Comment by crssi** [comment]
What did you find out, is it blocked or it is not now?

Off-Topic:
Why would someone NOT want to block `pubnub.com`?
Are there any breakages on IoT devices when blocked?

I am blocking it, a no IoT device doesn't show a problems here... and I see pubnub as tracking/spy or whatever unwanted stuff.

**Comment by vbap** [comment]
both pubnub and applovin are still being blocked as "Porn" category for me (oct 27). I don't mind that they are blocked, but I think the classification is incorrect.

**Comment by crssi** [comment]
I have enabled blocking "Porn" and have no problema with that.
Are you sure that is the reason?
Can you post a screenshot from log that says blocked by porn?

**Comment by vbap** [comment]
I have enabled blocking for Porn and Privacy. I have not enabled any blocklists or tracking blocks.
![image](https://user-images.githubusercontent.com/22019943/97287606-e621c300-1898-11eb-8c53-4085276199b4.png)

**Comment by crssi** [comment]
> both pubnub and applovin are still being blocked as "Porn"

`applvn.com` is not the same as `applovin.com`, your reporting is false since this topic is about `pubnub.com` and `applovin.com`

and what about screenshot of `pubnub.com` being blocked by `porn` category

**Comment by vbap** [comment]
You're right. Apologies new glasses for me!
